react-app/src/components/Container/Accordion/index.tsx

```diff
@@ -17,26 +17,37 @@ AccordionItem.displayName = "AccordionItem";
 
 type AccordionTriggerProps = {
   className?: string; // Add className to prop type definition
+  showPlusMinus?: boolean;
 } & React.ComponentPropsWithoutRef<typeof AccordionPrimitive.Trigger>;
 
 const AccordionTrigger = React.forwardRef<
   React.ElementRef<typeof AccordionPrimitive.Trigger>,
   AccordionTriggerProps
->(({ className, children, ...props }, ref) => (
-  <AccordionPrimitive.Header className="flex">
-    <AccordionPrimitive.Trigger
-      ref={ref}
-      className={cn(
-        "flex flex-1 items-center justify-between py-4 font-medium transition-all hover:underline [&[data-state=open]>svg]:rotate-180",
-        className,
-      )}
-      {...props}
-    >
-      {children}
-      <ChevronDown className="h-4 w-4 shrink-0 transition-transform duration-200" />
-    </AccordionPrimitive.Trigger>
-  </AccordionPrimitive.Header>
-));
+>(({ className, showPlusMinus, children, ...props }, ref) => {
+  const animationClass = showPlusMinus ? "group" : "[&[data-state=open]>svg]:rotate-180";
+  return (
+    <AccordionPrimitive.Header className="flex">
+      <AccordionPrimitive.Trigger
+        ref={ref}
+        className={cn(
+          `flex flex-1 items-center justify-between py-4 font-medium transition-all hover:underline ${animationClass}`,
+          className,
```

Comment:
```suggestion
          "flex flex-1 items-center justify-between py-4 font-medium transition-all hover:underline",
          className,
          animationClass,
```

Just a suggestion to simplify things